src/Context/ImageActorContext.js

```diff
@@ -2,9 +2,12 @@ class ImageActorContext {
   // MultiscaleSpatialImage to be visualized
   image = null
 
-  // The rendered image / label image scale
+  // The target image scale
   renderedScale = null
```

Comment:
To be accurate with the naming here, should this be called `targetScale` and `loadedScale` -> `renderedScale` ?

Reply:
Good idea.  Changed.  It breaks the Bootstrap and Material GUIs though 🤦

Reply:
Thanks -- could you please address Bootstrap and Material after merge?

Reply:
Will do.